its/plugin/tests/src/test/java/com/sonar/it/php/PHPUnitTest.java

```diff
@@ -71,11 +70,11 @@ public void setCoverageReportPaths(String reportPaths) {
   }
 
   public BuildResult executeBuild() {
-    return orchestrator.executeBuild(BUILD);
+    return ORCHESTRATOR.executeBuild(BUILD);
```

Comment:
This test uses `Tests.ORCHESTRATOR ` directly, but others (e.g. PHPTest) uses following assignment:

```
public static OrchestratorExtension orchestrator = Tests.ORCHESTRATOR;
```

It will be good that all tests use it in the same way.

Reply:
Good catch, that's also maybe why this project was failing with some solutions.